runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -538,6 +606,55 @@ XpSnapshotSingle getSkillSnapshot(Skill skill)
 		return xpState.getSkillSnapshot(skill);
 	}
 
+	/**
+	 * Determines the combat xp modifier for a skill based on the current attack style and
+	 * equipped weapon type.
+	 *
+	 * @param skill       skill to determine modifier for
+	 * @return double value of the combat xp modifier
+	 */
+	private double getCombatXpModifier(Skill skill)
+	{
+		if (equippedWeaponType == TYPE_28)
```

Comment:
This definitely isn't right, since type 28 is unused. I've noted in a PR of mine (#16256) that [this type belonged to the Tumeken's Heka during beta testing](https://github.com/Nightfirecat/runelite/blob/weapontype-names/runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/WeaponType.java#L84-L87), and is not used on any obtainable item.

Reply:
Mod Ash has confirmed that `28` is "self-powering wands" and that it is not currently in use: https://twitter.com/JagexAsh/status/1622951836872310784

Reply:
Thanks for sharing. To be clear, I don't doubt `28` is unused but rather that `23` doesn't line up with the wiki.